src/sections/About.html

```diff
@@ -0,0 +1,4 @@
+<section class="m-8 lg:m-8 p-5 md:p-8 lg:p-32 font-outfit md:text-[2rem] lg:text-5xl">
```

Comment:
The padding seems to be too big on the mobile breakpoint. Maybe you can decrease it.